src/mmw/apps/modeling/tasks.py

```diff
@@ -436,3 +275,98 @@ def to_gms_file(mapshed_data):
     output.seek(0)
 
     return output
+
+
+@shared_task(throws=Exception)
+def analyze_nlcd(result, area_of_interest=None):
+    if 'error' in result:
+        raise Exception('[analyze_nlcd] {}'.format(result['error']))
+
+    pixel_width = aoi_resolution(area_of_interest) if area_of_interest else 1
+
+    histogram = {}
+    total_count = 0
+    categories = []
+
+    # Convert results to histogram, calculate total
+    for key, count in result.iteritems():
+        total_count += count
+        histogram[make_tuple(key[4:])] = count  # Change {"List(1)":5} to {1:5}
```

Comment:
Yes. NLCD values are taken as is. Soil values are re-mapped, leading to multiple counts with the same key. Those counts are summed up to have one value for each key. I'll see if I can add a comment explaining this in `analyze_soil`